style.css

```diff
@@ -743,3 +747,677 @@ button:active {
   background: #0416b6;
   color: white;
 }
+
+@media screen and (min-width: 768px) {
+  body {
+    background-color: #e5e5e5;
+    font-family: "Poppins", sans-serif;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+  }
+
+  header {
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+    display: flex;
+    flex: 1 1 auto;
+    text-align: center;
+    padding: 1em;
+    background-color: #fff;
+    justify-content: space-between;
+  }
```

Comment:
- Please, kindly increase the padding top to make your design close to that of the Figma design as much as you can

Reply:
Thank you, I will solve it right away.